backend/src/core/config.py

```diff
@@ -12,9 +12,18 @@ class Settings(BaseSettings):
     APP_NAME: str
     APP_VERSION: str
 
-    SECRET_KEY: str
-    ACCESS_TOKEN_EXPIRE_MINUTES: int = 1440
+    UPLOADS_PATH: DirectoryPath
+    TEMPLATES_PATH: DirectoryPath
+
     ALGORITHM: str = "HS256"
```

Comment:
you set these parameters in .env
```suggestion
    ALGORITHM: str
```

Reply:
they're more like default settings to simplify initial .env configuration

Reply:
It is better to remove this and configure it in .env